Братишка. Я тебе форматика принес.

JAremko opened the pull request:
Могу остальные поправить если нужно.

Commit:
oauth добавлено

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update LICENSE.md

JAremko committed:
Update LICENSE.md

JAremko committed:
Update README.md

artaommahe commented:
Отличный вариант, спасибо ) если есть время - допиливай остальное и мержнем, если нет - сами допилим )

JAremko committed:
Update admin.md

JAremko committed:
Update LICENSE.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko commented on chat.md:
````diff
@@ -38,42 +61,48 @@ socket.On(Socket.EVENT_CONNECT, (a) =>
+#####URL:[`/chat/history`](http://funstream.tv/api/chat/history)  
+```js
 {
     channel: <string|null> channel id or null for default
````
Переведу как имя а не Идентификатор канала.

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko commented:
В `/chat/message` `channel: <int> channel id`,  а в `/chat/history` `channel: <string|null> channel id` Во втором это имя, а в первом Идентификатор?  И где его брать, этот идентификатор? Если я конечно это все правильно понял. Наверное сноска не помешает.

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update admin.md

JAremko committed:
Update admin.md

JAremko committed:
Merge remote-tracking branch 'funstream-api/master'

Conflicts:
	chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Update chat.md

JAremko committed:
Реорганизация.

С такой структурой репозитория будет гораздо легче расширять API. Также
можно будет хранить примеры и утилиты относящиесь к конкретному API в
его разделе.

JAremko committed:
Реорганизация.

С такой структурой репозитория будет гораздо легче расширять API. Также
можно будет хранить примеры и утилиты относящиесь к конкретному API в
его разделе.

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Delete admin.md

JAremko committed:
Delete admin.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create Node.js

JAremko committed:
Create Node.js

JAremko committed:
Create CSharp.cs

JAremko committed:
Create CSharp.cs

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Create protocol.md

JAremko committed:
Create protocol.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Реструктуризация.

JAremko committed:
Реструктуризация.

JAremko committed:
Create channels.MD

JAremko committed:
Create channels.MD

JAremko committed:
Rename channels.MD to channels.md

JAremko committed:
Rename channels.MD to channels.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Delete README.md

JAremko committed:
Delete README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update chat.md

JAremko committed:
Delete chat.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Create README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Delete oauth.md

JAremko committed:
Delete oauth.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko committed:
Update README.md

JAremko commented:
Наверное можно маржить. Разбор common.md займет прилично времени, боюсь что если сейчас не замаржить то придется ребейсы делать когда вы будите документацию обновлять.

artaommahe commented:
Угу, только ребейзни всю эту кучу в 1 коммит, чтоб не было в общей хистори десятков коммитов по одному изменению ) и мержнем

JAremko committed:
temp

JAremko committed:
Merge remote-tracking branch 'origin/master'

Conflicts:
	chat.md
